waspc/README.md

```diff
@@ -234,6 +234,8 @@ Not only that, but Haskell build tools don't have a good support for mixing them
 Tests are run with `stack test`.
 You can do `stack test --coverage` to see the coverage.
 
+To run individual test, you can do `stack test --test-arguments "-p \"Some test description to match\""`.
+
```

Comment:
Hah yes, I put it here because it took me 15 minutes to figure out how to do it :D. Tests themselves print instructions on how to do it but those don't work.